elements/bulbs-pinned-element/bulbs-pinned-element.js

```diff
@@ -1,34 +1,46 @@
-import { registerElement, BulbsHTMLElement } from 'bulbs-elements/register';
 import invariant from 'invariant';
-import { getScrollOffset, InViewMonitor } from 'bulbs-elements/util';
+
```

Comment:
@collin @MelizzaP @shawncook switched up the `bulbs-pinned-element` to move the content to the inside of a `car` on a `rail` and move only in respect to that `rail`. My assumption is that the rail will always match up with the content height that the rail should be following. This gives us a bit more control over how the pinned element's container interacts with the content inside of it.

Working on tests now, but in the meantime please review my `bulbs-pinned-element` changes in isolation from the rest of the PR. Let me know, at least conceptually, if you agree with these changes or want to use a different strategy.